.nvmrc

```diff
@@ -1 +1 @@
-12.13.1
\ No newline at end of file
+18.18.2
```

Comment:
Out of interest, why upgrade to Node 18 over 20? Was this because of issues with `whatwg-fetch` and `url.parse()`? In our case would this [throw errors rather than just warnings](https://github.com/nodejs/node/pull/45526)?

Reply:
Good point, have bumped up to 20 now! 18 was the first LTS version I started having problems with so I was focused on getting that running, it seems that 20 is working now too.